kanidmd_web_ui/src/login.rs

```diff
@@ -262,7 +263,7 @@ impl Component for LoginApp {
     type Properties = ();
 
     fn create(_: Self::Properties, link: ComponentLink<Self>) -> Self {
-        ConsoleService::log(format!("create").as_str());
+        ConsoleService::log(&"create".to_string());
```

Comment:
Should just be "create" no need to & and to_string here.

Reply:
done